index.src.html

```diff
@@ -3745,29 +3748,41 @@
 
 : <dfn method for=VideoFrame>copyTo(|destination|, |options|)</dfn>
 :: Asynchronously copies the planes of this frame into |destination| according
-    to |options|. The format of the data is the same as this {{VideoFrame}}'s
-    {{VideoFrame/format}}.
+    to |options|. The format of the data is specified by {{VideoFrameCopyToOptions/format}},
+    if present, otherwise {{VideoFrame/format}}.
```

Comment:
When looking at how this is rendered in HTML, I find it very confusing, can you rephrase it or make it more explicit or something?